### PR TITLE
Added min shuffle out percentage

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added min shuffle out percentage value
- adjusted the yearly inflation rate values

### CODE REVIEW

File: cmd/node/main.go [hunk 1]
```diff
@@ -1796,15 +1798,26 @@ func createNodesCoordinator(
...
+	minDurationBeforeStopProcess := int64(minThresholdEpochDuration * float64(epochDuration))
+	maxDurationBeforeStopProcess := int64(maxThresholdEpochDuration * float64(epochDuration))
+
+	minDurationInterval := time.Millisecond * time.Duration(minDurationBeforeStopProcess)
 	maxDurationInterval := time.Millisecond * time.Duration(maxDurationBeforeStopProcess)
-	minDurationInterval := maxDurationInterval / 2
-	//waiting interval will be [maxDuration/2 and maxDuration]
+
+	log.Debug("closing.NewShuffleOutCloser",
+		"maxDurationInterval", minDurationInterval,
```
Comment (bogdan-rosianu) [inline cmd/node/main.go:1818]: fix log

Comment (bogdan-rosianu) [reply cmd/node/main.go:1818]: you don't use minDuration else where?

File: cmd/node/main.go [hunk 2]
```diff
@@ -1796,15 +1798,26 @@ func createNodesCoordinator(
...
+	if !(maxThresholdEpochDuration >= 0.0 && maxThresholdEpochDuration <= 1.0) {
+		return nil, nil, fmt.Errorf("invalid max threshold for shuffled out handler")
 	}
-	maxDurationBeforeStopProcess := int64(nodesConfig.RoundDuration) * epochConfig.RoundsPerEpoch
-	maxDurationBeforeStopProcess = int64(thresholdEpochDuration * float64(maxDurationBeforeStopProcess))
+	minThresholdEpochDuration := epochConfig.MinShuffledOutRestartThreshold
+	if !(minThresholdEpochDuration >= 0.0 && minThresholdEpochDuration <= 1.0) {
+		return nil, nil, fmt.Errorf("invalid min threshold for shuffled out handler")
+	}
+
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply cmd/node/main.go:1809]: done inside NewShuffleOutCloser. Added test

File: cmd/node/main.go (hunk 1 above)
Comment (iulianpascalau) [reply cmd/node/main.go:1818]: log fixed. The constructor uses both minDurationInterval and maxDurationInterval

**Pull request merged**